src/posix.mak

```diff
@@ -486,18 +486,12 @@ endif
 
 ######## generate a default dmd.conf
 
-define DEFAULT_DMD_CONF
-[Environment32]
-DFLAGS=-I%@P%/../../../../../druntime/import -I%@P%/../../../../../phobos -L-L%@P%/../../../../../phobos/generated/$(OS)/$(BUILD)/32$(if $(filter $(OS),osx),, -L--export-dynamic)
-
-[Environment64]
-DFLAGS=-I%@P%/../../../../../druntime/import -I%@P%/../../../../../phobos -L-L%@P%/../../../../../phobos/generated/$(OS)/$(BUILD)/64$(if $(filter $(OS),osx),, -L--export-dynamic) -fPIC
-endef
-
-export DEFAULT_DMD_CONF
+$G/makedmdconf: ../makedmdconf.d $(HOST_DMD_PATH)
+	@echo "  (HOST_DMD_RUN)  $<  $<"
+	$(HOST_DMD_RUN) $< -of$@
 
-$G/dmd.conf: $(SRC_MAKE)
-	echo "$$DEFAULT_DMD_CONF" > $@
+$G/dmd.conf: $G/makedmdconf FORCE
```

Comment:
FORCE is a bad Makefile style, but apart from that: Why would we want to regenerate it all the time?

Reply:
Not generate it all the time, just call the tool.  The tool checks whether or not it needs to be updated.